operators/pkg/tenant-controller/tenant_controller.go

```diff
@@ -307,6 +347,72 @@ func (r *TenantReconciler) checkValidWorkspaces(ctx context.Context, tn *crownla
 	return tenantExistingWorkspaces, workspaces, err
 }
 
+// deleteClusterNamespace deletes the namespace for the tenant, if it fails then it returns an error
+func (r *TenantReconciler) deleteClusterNamespace(ctx context.Context, tn *crownlabsv1alpha2.Tenant, nsName string) (nsOk bool, err error) {
```

Comment:
The nsOk return value is redundant, as already expressed by the presence of the error.

Reply:
Resolved in [a6ef668](https://github.com/netgroup-polito/CrownLabs/pull/821/commits/a6ef668a82445161f21d0562c19f5be5e6064621)